src/Compilers/VisualBasic/Portable/Lowering/LocalRewriter/LocalRewriter_Conversion.vb

```diff
@@ -96,10 +96,24 @@ Namespace Microsoft.CodeAnalysis.VisualBasic
             End If
 
             If node.RelaxationLambdaOpt IsNot Nothing Then
-                returnValue = node.Update(VisitExpressionNode(node.RelaxationLambdaOpt),
+
+                If _inExpressionLambda AndAlso
+                    NoParameterRelaxation(DirectCast(node.Operand, BoundLambda).LambdaSymbol, node.RelaxationLambdaOpt.LambdaSymbol) Then
```

Comment:
I think this should work for getting an instance of an Anonymous Delegate
```
Dim anonymousDelegate = <lambda>
```


---
In reply to: [64994198](https://github.com/dotnet/roslyn/pull/11627#discussion_r64994198) [](ancestors = 64994198,64994158)